# fix skiping of "" in word[] array.
# Side effect: fixes DCC RESUME with mIRC >= 6.1
--- xchat-2.0.5/src/common/outbound.c	2003-09-05 21:12:52.000000000 +1000
+++ xchat-2.0.6/src/common/outbound.c	2003-10-16 14:00:52.000000000 +1000
@@ -185,8 +185,10 @@
 			if (!handle_quotes)
 				goto def;
 			if (quote)
+			{
 				quote = FALSE;
-			else
+				space = FALSE;
+			} else
 				quote = TRUE;
 			cmd++;
 			break;
